spring-boot-jib/src/main/java/com/baeldung/GreetingController.java

```diff
@@ -0,0 +1,20 @@
+package com.baeldung;
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.concurrent.atomic.AtomicLong;
+
+@RestController
+public class GreetingController {
+
+    private static final String template = "Hello Docker, %s!";
+    private final AtomicLong counter = new AtomicLong();
+
+    @RequestMapping("/greeting")
```

Comment:
This should be `@GetMapping` instead. It's a bit simpler to read and it also doesn't expose methods unintentionally.